shell/packages/sandstorm-db/db.js

```diff
@@ -2394,8 +2424,8 @@ if (Meteor.isServer) {
     check(intervalMs, Number);
     check(callback, Function);
 
-    if (intervalMs < 120000) {
-      throw new Error("less than 2-minute cleanup interval seems too fast; " +
+    if (intervalMs < 45000) {
+      throw new Error("less than 45-second cleanup interval seems too fast; " +
```

Comment:
Do we really want to be polling for jobs every minute, or do we maybe want to set `minimumSchedulingSlack` to something higher?

Reply:
I'm fine with just making the slack larger. Maybe 5 minutes?

Reply:
SGTM. If it turns out to be a problem we can always reduce it later without breaking anything.